code/index.html

```diff
@@ -1,30 +1,190 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Hay Harvest</title>
+  <link rel="stylesheet" href="./style.css">
+  <!-- dont forget to add a css file and link it here! -->
+</head>
+
+<body>
+  <!-- video or image as a header is cool :) -->
+  <header>
+    <h1>KÄLLSLÄTTEN <br> HAY HARVEST</h1>
```

Comment:
Instead of using the `<br>`, which is considered bad practise, try to use CSS styling or divide it into two h1-tags!